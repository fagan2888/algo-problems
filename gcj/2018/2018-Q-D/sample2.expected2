Case #1: 
0.5000000000 0.0000000000 0.0000000000
0.0000000000 0.5000000000 0.0000000000
0.0000000000 0.0000000000 0.5000000000
Case #2: 
0.4971960144 0.0528039856 0.0028039856
-0.0528039856 0.4943920289 0.0528039856
0.0028039856 -0.0528039856 0.4971960144
Case #3: 
0.4857023073 0.1178509427 0.0142976927
-0.1178509427 0.4714046145 0.1178509427
0.0142976927 -0.1178509427 0.4857023073
Case #4: 
0.4857022604 0.1178511302 0.0142977396
-0.1178511302 0.4714045208 0.1178511302
0.0142977396 -0.1178511302 0.4857022604
Case #5: 
0.4770620726 0.1479379274 0.0229379274
-0.1479379274 0.4541241452 0.1479379274
0.0229379274 -0.1479379274 0.4770620726
Case #6: 
0.4040969794 0.2784030206 0.0959030206
-0.2784030206 0.3081939588 0.2784030206
0.0959030206 -0.2784030206 0.4040969794
