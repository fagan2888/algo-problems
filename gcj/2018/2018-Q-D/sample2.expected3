Case #1: 
0.5000000000 0.0000000000 0.0000000000
0.0000000000 0.5000000000 0.0000000000
0.0000000000 0.0000000000 0.5000000000
Case #2: 
0.4971722949 -0.0528039856 0.0056078819
0.0531009344 0.4943920289 -0.0525053574
0.0000000000 0.0528039856 0.4972039211
Case #3: 
0.4850713011 -0.1178509427 0.0285830039
0.1212676083 0.4714046145 -0.1143322203
0.0000000000 0.1178509427 0.4859127034
Case #4: 
0.4850712501 -0.1178511302 0.0285830975
0.1212678125 0.4714045208 -0.1143323901
0.0000000000 0.1178511302 0.4859126579
Case #5: 
0.4754099598 -0.1479379274 0.0458229176
0.1548721090 0.4541241452 -0.1406623282
0.0000000000 0.1479379274 0.4776132009
Case #6: 
0.3710307903 -0.2784030206 0.1866223748
0.3351658584 0.3081939588 -0.2065921855
0.0000000000 0.2784030206 0.4153212710
