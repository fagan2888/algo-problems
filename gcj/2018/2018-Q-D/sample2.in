6
1.000000
1.2
1.414213
1.4142135623730951
1.5
1.73
