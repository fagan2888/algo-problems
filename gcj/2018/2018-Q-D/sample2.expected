Case #1: 
0.3535533906 0.0000000000 0.3535533906
0.0000000000 0.5000000000 -0.0000000000
-0.3535533906 0.0000000000 0.3535533906
Case #2: 
0.3535533906 0.0528039856 0.3495879562
0.0000000000 0.4943920289 -0.0746761125
-0.3535533906 0.0528039856 0.3495879562
Case #3: 
0.3535533906 0.1178509427 0.3333333996
0.0000000000 0.4714046145 -0.1666664016
-0.3535533906 0.1178509427 0.3333333996
Case #4: 
0.3535533906 0.1178511302 0.3333333333
0.0000000000 0.4714045208 -0.1666666667
-0.3535533906 0.1178511302 0.3333333333
Case #5: 
0.3535533906 0.1479379274 0.3211142626
0.0000000000 0.4541241452 -0.2092158233
-0.3535533906 0.1479379274 0.3211142626
Case #6: 
0.3535533906 0.2784030206 0.2179260382
0.0000000000 0.3081939588 -0.3937213275
-0.3535533906 0.2784030206 0.2179260382
